packages/components/src/components/segmented-control/segmented-control/segmented-control.tsx

```diff
@@ -33,7 +37,7 @@ export class SegmentedControl {
   @Element() public host!: HTMLElement;
 
   /** Background color variations */
-  @Prop() public backgroundColor?: SegmentedControlBackgroundColor = 'background-default';
+  @Prop() public backgroundColor?: SegmentedControlBackgroundColor; // 'background-color' prop is deprecated
```

Comment:
When entire prop is deprecated, it would make sense to apply the `@deprecated` annotation in the jsdoc comment:
```
/** @ deprecated 
Background color variations */
```

Reply:
Done